src/benchmarks/micro/libraries/Microsoft.Extensions.DependencyInjection/TimeToFirstServiceBenchmark.cs

```diff
@@ -2,6 +2,11 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+// Define INTERNAL_DI on platforms that support emit.
+#define INTERNAL_DI
```

Comment:
> To run with the different engines, uncomment the define.

NIT: Maybe you were intending to comment it by default?
```suggestion
//#define INTERNAL_DI
```